codestodians/komlie-api
# [Feature] Rework cart product model relationship

#### What does this PR do?
- Reworks the relationship table between the cart and product model to account for item quantity

#### Description of Task to be completed?
- Create `Cart_Product` model to wrestle rights for tables joining from django
- Refactor cart operations logic to account for quantity manipulation
- Add new fixture
- Update tests

#### How should this be manually tested?
- Checkout into the `feature/rework-cart-product-model-joining` branch
- Run migration with `python3 manage.py migrate`
- Run the test suites and/or specific tests on `GraphiQL` to ensure it works perfectly.

#### Any background context you want to provide?
- The backend needed to account for item quantity in cart, hence, this became necessary.

#### What are the relevant pivotal tracker stories?
- [#167086967](https://www.pivotaltracker.com/story/show/167086967)
